From 5a840d475b16354777c6c39d1a956df7a790564e Mon Sep 17 00:00:00 2001
From: MengLi <meng.li@windriver.com>
Date: Tue, 3 Jul 2018 14:06:33 +0800
Subject: [PATCH 1333/1345] arm64: dts: marvell: add needs-reset-on-resume
 property in usb dts node

Add needs-reset-on-resume to force xhci reset after resume.
Detail reason, please refer to SDK kernel commit 9154dabd1d84
("dts: cp110: force xhci reset after resume")

Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../boot/dts/marvell/armada-cp110-master.dtsi      |    2 ++
 .../arm64/boot/dts/marvell/armada-cp110-slave.dtsi |    2 ++
 2 files changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/marvell/armada-cp110-master.dtsi b/arch/arm64/boot/dts/marvell/armada-cp110-master.dtsi
index 3ac2890..8120ba8 100644
--- a/arch/arm64/boot/dts/marvell/armada-cp110-master.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-cp110-master.dtsi
@@ -156,6 +156,7 @@
 				dma-coherent;
 				interrupts = <ICU_GRP_NSR 106 IRQ_TYPE_LEVEL_HIGH>;
 				clocks = <&cpm_syscon0 1 22>;
+				needs-reset-on-resume;
 				status = "disabled";
 			};
 
@@ -166,6 +167,7 @@
 				dma-coherent;
 				interrupts = <ICU_GRP_NSR 105 IRQ_TYPE_LEVEL_HIGH>;
 				clocks = <&cpm_syscon0 1 23>;
+				needs-reset-on-resume;
 				status = "disabled";
 			};
 
diff --git a/arch/arm64/boot/dts/marvell/armada-cp110-slave.dtsi b/arch/arm64/boot/dts/marvell/armada-cp110-slave.dtsi
index e054ef0..cdd9bdb 100644
--- a/arch/arm64/boot/dts/marvell/armada-cp110-slave.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-cp110-slave.dtsi
@@ -117,6 +117,7 @@
 				dma-coherent;
 				interrupts = <ICU_GRP_NSR 106 IRQ_TYPE_LEVEL_HIGH>;
 				clocks = <&cps_syscon0 1 22>;
+				needs-reset-on-resume;
 				status = "disabled";
 			};
 
@@ -127,6 +128,7 @@
 				dma-coherent;
 				interrupts = <ICU_GRP_NSR 105 IRQ_TYPE_LEVEL_HIGH>;
 				clocks = <&cps_syscon0 1 23>;
+				needs-reset-on-resume;
 				status = "disabled";
 			};
 
-- 
1.7.9.5

